R/normdm.R

```diff
@@ -0,0 +1,51 @@
+#' normdm
+#'
+#' @param k number of groups to be simulated
+#' @param names names for each group
```

Comment:
this and succeeding parameters should be written in the following form:

a vector of character strings representing group names
a vector of n values representing group sample sizes
etc